var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -30,16 +30,17 @@ class SuiteSparse(Package):
     SuiteSparse is a suite of sparse matrix algorithms
     """
     homepage = 'http://faculty.cse.tamu.edu/davis/suitesparse.html'
-    url = 'http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-4.5.1.tar.gz'
+    url = 'http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-4.5.1.tar.gz'  # noqa: E501
```

Comment:
Is there a config file in the repo that should be disabling this warning? Otherwise, locally, I see:

```
 oxberry1 > … > builtin > packages > suite-sparse$ flake8 package.py
package.py:33:80: E501 line too long (82 > 79 characters)
package.py:41:80: E501 line too long (122 > 79 characters)
(snip)
```